schema.go

```diff
@@ -106,3 +113,28 @@ func findBuildByBuilder(db *mgo.Database, builder string, buildnum int) (*LogKee
 	}
 	return build, nil
 }
+
+func GetOldTests(db *mgo.Database, now time.Time) (*[]Test, error) {
+	query := bson.M{
+		"started": bson.M{"$lte": now.Add(-deletePassedTestCutoff)},
+	}
+	tests := []Test{}
+	err := db.C("tests").Find(query).Sort("-started").Limit(maxTests).All(&tests)
+	if err != nil {
+		return nil, errors.Wrap(err, "error finding tests")
+	}
+	return &tests, err
```

Comment:
don't return pointers to slices, slices are basically already pointers so this isn't needed, and you can return nil for a slice.